assets/javascripts/lib/discourse-markdown/bbcode-plugin.js

```diff
@@ -87,6 +87,7 @@ export function setup(helper) {
     "span.bb-inline-spoiler",
     "span.bb-highlight",
     "span.bb-pindent",
+    "span.bb-header",
```

Comment:
Can you alphebetise this? Should be 
```
"span.bb-inline-spoiler",
"span.bb-header",
"span.bb-highlight",
```

Reply:
Correction made! Question. What exactly is the custom for alphabetization? I notice that the bb-inline is above bb-header. Is that just an uncaught slip or is there a reason for that?